.github/workflows/advanced-on-demand.yml

```diff
@@ -54,6 +54,12 @@ jobs:
         with:
           poetry-version: ${{ matrix.poetry-version }}
 
+      - name: Install LLVM and Clang
```

Comment:
can you add a comment on why we need LLVM and clang? This may help in the future when someone is trying to decide whether to keep it or not? If comments are not allowed, maybe we can just add it to the name.

Reply:
Added a comment, need it for scipy in Python 3.12. Would hope this changes in the future.